src/docker/deployer/sh/build-publish-payloads.sh

```diff
@@ -31,6 +31,15 @@ aptos move build-publish-payload \
 	--skip-fetch-latest-git-deps \
 	--move-2
 
+aptos move build-publish-payload \
+	--assume-yes \
+	--named-addresses \
+	emojicoin_dot_fun=$profile,market_metadata=$profile \
+	--package-dir $move_dir/market_metadata/ \
+	--json-output-file $json_dir/market_metadata.json \
+	--skip-fetch-latest-git-deps \
+	--move-2
```

Comment:
```suggestion
	--skip-fetch-latest-git-deps
```

(Move 2 is now default I believe)